rosbridge_library/src/rosbridge_library/internal/message_conversion.py

```diff
@@ -91,7 +89,7 @@
     "string",
 ]
 ros_header_types = ["Header", "std_msgs/Header", "roslib/Header"]
-ros_binary_types = ["uint8[]", "char[]"]
+ros_binary_types = ["uint8[]", "char[]", "sequence<uint8>", "sequence<char>"]
```

Comment:
Since `uint8[]` will be `sequence<uint8>`, I marked it as `binary`.
https://design.ros2.org/articles/idl_interface_definition.html

I think I can drop this, but then `sensor_msgs/msg/Image` won't be `binary` as well.
https://github.com/ros2/common_interfaces/blob/f41462f5dd36f5c3ae28866f18f356423ffb84e4/sensor_msgs/msg/Image.msg#L26